.gitignore

```diff
@@ -34,3 +34,5 @@ CMakeLists.txt.user
 *.kdev4
 # Ignore IDEA/Clion files/dirs
 /.idea/
+# Ignore Visual Studio Code's working dir
+.vscode/
```

Comment:
If Visual Studio Code only creates this directory at the top-level, please add a leading slash.